templates/ceph/managed.html

```diff
@@ -0,0 +1,255 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+
+{% block title %}Managed Ceph | Ceph{% endblock %}
+
+{% block meta_description %}Canonical is the leading managed ceph provider, we build, support and manage ceph clusters in any location. {% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-7">
+      <h1>Fully managed Ceph</h1>
+      <p>Reduce operational complexity by allowing Canonical's experts to operate and maintain your Ceph cluster.</p>
+      <ul class="p-list">
+				<li class="p-list__item is-ticked">Fully managed on your servers, in your own data center, or in a co-location facility</li>
+				<li class="p-list__item is-ticked">Built using a proven reference architecture</li>
+				<li class="p-list__item is-ticked">Customise the cluster architecture based on your needs</li>
+				<li class="p-list__item is-ticked">Integrated with your infrastructure, on-prem, hybrid, or public cloud</li>
+				<li class="p-list__item is-ticked">Existing workload analysis and migration services</li>
+				<li class="p-list__item is-ticked">Enterprise SLA for all protocols</li>
+				<li class="p-list__item is-ticked">Your team retains access to all machines at all times</li>
+				<li class="p-list__item is-ticked">Modern monitoring and log management</li>
+				<li class="p-list__item is-ticked">We will hand over the keys when you are ready</li>
+      </ul>
```

Comment:
Sorry didn't spot this first time round - you're missing this paragraph here "Focus on your business applications, while Canonical takes care of your storage needs."

Reply:
Fixed! Thanks Beth!